doc/build-unix.md

```diff
@@ -67,15 +67,18 @@ Build requirements:
 
     sudo apt-get install build-essential libtool autotools-dev automake pkg-config libssl-dev libevent-dev bsdmainutils
 
-On at least Ubuntu 14.04+ and Debian 7+ there are generic names for the
+
+Options when installing required Boost library files:
+
+1. On at least Ubuntu 14.04+ and Debian 7+ there are generic names for the
 individual boost development packages, so the following can be used to only
 install necessary parts of boost:
 
-    sudo apt-get install libboost-system-dev libboost-filesystem-dev libboost-chrono-dev libboost-program-options-dev libboost-test-dev libboost-thread-dev
+    `sudo apt-get install libboost-system-dev libboost-filesystem-dev libboost-chrono-dev libboost-program-options-dev libboost-test-dev libboost-thread-dev`
```

Comment:
Thanks Jonas.
The Ordered list (numbers 1 and 2) formatting cause markdown to ignore the 4-space "code" indentation markup.  I did [some experimentation](https://github.com/jonathancross/my-docs/blob/master/Github%20markdown%20list%20and%20code.md) and now see that I need to indent 8 spaces to bring back the code formatting when in a list.